nimare/meta/kernel.py

```diff
@@ -169,8 +169,10 @@ def transform(self, dataset, masker=None, return_type="image"):
                 img = kernel_data[mask_data]
                 imgs.append(img)
             elif return_type == "image":
+                hdr = nib.Nifti1Header()  # Passing an int64 array to Nifti1Image requires a header
+                hdr.set_data_dtype(np.int64)
                 kernel_data *= mask_data
-                img = nib.Nifti1Image(kernel_data, mask.affine)
+                img = nib.Nifti1Image(kernel_data, mask.affine, header=hdr)
```

Comment:
Since this is the general KernelTransformer, and there are a kernels that won't produce int data (e.g., the ALE kernel), I think it might be better to coerce the data in associated kernel transformers (KDA, MKDA) to int32.